src/raw/connection.rs

```diff
@@ -48,26 +48,26 @@ mod tokio_connection {
             let fd = tokio::fs::OpenOptions::new()
                 .write(true)
                 .read(true)
+                .custom_flags(libc::O_NONBLOCK)
                 .open(DEV_FUSE)
                 .await?
                 .into_std()
                 .await
                 .into_raw_fd();
 
-            Self::set_fd_non_blocking(fd)?;
-
             Ok(Self {
                 fd: AsyncFd::new(fd)?,
                 read: Mutex::new(()),
                 write: Mutex::new(()),
             })
         }
 
-        #[cfg(feature = "unprivileged")]
+        #[cfg(all(target_os = "linux", feature = "unprivileged"))]
         pub async fn new_with_unprivileged(
             mount_options: MountOptions,
             mount_path: impl AsRef<Path>,
         ) -> io::Result<Self> {
+
```

Comment:
Could you remove this blank line? Because It seems there is no reason to exist for this blank line